dap_aria_mapping/pipeline/entity_processing/tests/test_reformat_entities.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+
+from dap_aria_mapping.pipeline.entity_processing.reformat_entities import (
```

Comment:
would need to be-formatted slightly given the slight difference in new entities sam's given us